packages/taquito-sapling/src/sapling-transaction-viewer.ts

```diff
@@ -1,23 +1,275 @@
-import * as sapling from '@airgap/sapling-wasm'
+import * as sapling from '@airgap/sapling-wasm';
+import BigNumber from 'bignumber.js';
+import { hex2buf, mergebuf } from '@taquito/utils';
+import { CommitmentsAndCiphertexts, RpcClient, RpcClientInterface } from '@taquito/rpc';
+import blake from 'blakejs';
+import { openSecretBox } from '@stablelib/nacl';
+import { InMemoryViewingKey } from './in-memory-viewing-key';
+import { bufToUint8Array, convertValueToBigNumber, readableFormat } from './helpers';
 
+const KDF_KEY = 'KDFSaplingForTezosV1';
```

Comment:
should we make a `constants.ts` file to house these? Just in case more constants are introduced. There's a uncommittedMerkleHash constant byte string that could live there as well

Reply:
Sure, we can create a constants.ts file